pages/class/[classid].tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useContext } from "react";
+import type { NextApplicationPage } from "../_app";
+import { AuthContext } from "../../context/AuthContext";
+import styles from "../../styles/class.module.css";
+import { Error } from "../../components/util/Error";
+import { Roster } from "../../components/Roster/Roster";
+import { useRouter } from "next/router";
+import { Attendance } from "../../components/Attendance/Attendance";
+import { Module } from "../../components/ModuleView/Module";
+import useSWR from "swr";
+import { APIContext } from "../../context/APIContext";
+
+const BackButton: React.FC = () => {
+  const router = useRouter();
+  return (
+    <>
+      <img src="back_button.svg" onClick={() => router.push("/home")} alt="back" className={styles.backbutton} />
+    </>
+  );
+};
+
+const Class: NextApplicationPage = () => {
+  const router = useRouter();
+  const client = useContext(APIContext);
+
+  const { user } = useContext(AuthContext);
+  const { classid } = router.query;
```

Comment:
this should be `const classId = router.query.classId as string`

Reply:
Meaning, ypu also need to change the query param to `classId` (so change the folder name to that)